dali/ft/filecopy.cpp

```diff
@@ -3136,12 +3131,17 @@ void FileSprayer::updateTargetProperties()
                      ((stricmp(aname,FArecordCount)==0)&&!gotrc) ||
                      ((stricmp(aname,"@blockCompressed")==0)&&copyCompressed) ||
                      ((stricmp(aname,"@rowCompressed")==0)&&copyCompressed)||
-                     (stricmp(aname,"@local")==0)
+                     (stricmp(aname,"@local")==0)||
+                     (stricmp(aname,"@checkSum")==0)||
```

Comment:
Which makes me more dubious as to whether copying @checkSum unconditionally here is correct.

copying source @checkSum isn't going to be correct if file changed at all.